src/ui/session/content/message_row/bubble.rs

```diff
@@ -213,18 +217,16 @@ impl MessageBubble {
 
         self.remove_css_class("outgoing");
 
-        if let Some(binding) = imp.sender_binding.take() {
-            binding.unwatch();
-        }
+        imp.sender_label
+            .set_label(&sponsored_message.sponsor_label());
+
+        let mut s = DefaultHasher::new();
+
+        sponsored_message.sponsor_label().hash(&mut s);
```

Comment:
Just a bit refactoring (removed blank lines and rename s to hasher).
```suggestion
        let mut hasher = DefaultHasher::new();
        sponsored_message.sponsor_label().hash(&mut hasher);
        let label_hash = hasher.finish() as i64;
```

Reply:
the variable `s` is also used in other places, like here:
https://github.com/paper-plane-developers/paper-plane/blob/62902f5f3f651c15fa98f7f4245766bf59024abb/src/ui/session/content/message_row/bubble.rs#L276-L278
Should I also rename there?

Reply:
O yes. Missed that. You could also move the hashing logic into its own function.

Reply:
Which would be the best place to make this kind of functions? bubble.rs (since it's only used there)?

Reply:
Yes, I don't think we will use it anywhere else. But if, we can then move i to `utils`.